pkg/s3/utils.go

```diff
@@ -12,15 +14,19 @@ import (
 
 var isEC2Instance bool
 var isEC2InstanceOnce sync.Once
+var isEC2InstanceTimeout = 2 * time.Second
```

Comment:
perhaps this could be configured via and env var in an init method? We/Users may not always want a timeout of 2 seconds.

Reply:
this is dead code. will remove it